<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Посадочная платформа</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Загрузки/mai.gif</normaloff>../../../../Загрузки/mai.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="listCOM"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_UPD_COM">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Загрузки/Схема стенда.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>464</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_A">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Толкатель A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_A1">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_A2">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_B">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Толкатель B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_B1">
         <property name="text">
          <string>▼</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_B2">
         <property name="text">
          <string>▲</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_C">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Толкатель C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_C1">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_C2">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_D">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Толкатель D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_D1">
         <property name="text">
          <string>▼</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_D2">
         <property name="text">
          <string>▲</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Загрузки/Схема стенда.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Количество шагов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="listCOM_Steps"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_Z">
       <property name="text">
        <string>Возврат в 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_NZ">
       <property name="text">
        <string>Устанвоить 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="closeCOM">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openCOM">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
